### PR TITLE
Add AWSSecretsManagerGetSecretValueAccess Policy

### PR DESCRIPTION
**Pull request by willtong1234**
*Description of changes:*

This change adds a AWSSecretsManagerGetSecretValueAccess policy which grants access to the Secrets Manager GetSecretValue API for a single secret ARN.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/policy_templates_data/policy_templates.json [hunk 1]
```diff
@@ -931,6 +931,34 @@
...
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "secretsmanager:GetSecretValue"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "${secretArn}",
```
Comment (jlhood) [inline docs/policy_templates_data/policy_templates.json]: Our policy templates follow the pattern of forcing scoping the resource to the current account and region for tighter security. Will following this pattern work for your use case? For example, you could change the SecretArn parameter to SecretName. Then do this:

```
"Resource": {
  "Fn::Sub": [
    "arn:${AWS::Partition}:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:${secretName}",
    {
      "secretName": {
        "Ref": "SecretName"
      }
    }
  ]
}
```

File: docs/policy_templates_data/policy_templates.json [hunk 2]
```diff
@@ -931,6 +931,34 @@
         ]
       }
     },
+    "AWSSecretsManagerGetSecretValueAccess": {
```
Comment (jlhood) [inline docs/policy_templates_data/policy_templates.json]: The other policy templates end in Policy. How about `AWSSecretsManagerGetSecretValuePolicy`?

File: docs/policy_templates_data/policy_templates.json (hunk 1 above)
Comment (jlhood) [reply docs/policy_templates_data/policy_templates.json]: Discussed use case offline. There is a use case for needing to specify the full ARN.

**Comment by brettstack** [comment]
This file no longer exists on the develop branch as it was redundant. Please rebase and make your changes to https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/policy_templates_data/policy_templates.json instead.

**Comment by willtong1234** [comment]
This is eclipsed by https://github.com/awslabs/serverless-application-model/pull/539. Closing this one out.

**Pull request closed without merging**